src/tests/codeCoverage.ts

```diff
@@ -153,7 +179,7 @@ export class CodeCoverage {
   }
 
   @elapsedTime()
-  private async queryPerClassCodeCov(
+  public async queryPerClassCodeCov(
```

Comment:
What change to public?

Reply:
Because the function is [stubbed](https://github.com/forcedotcom/salesforcedx-apex/blob/4b35861d0199e9e65891886dd1774a8d9952f803/test/tests/codeCoverage.test.ts#L166) here in the unit test in order to mock the exception.